src/it/casperjs-runner/integration-test/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.github.casperjs</groupId>
+  <artifactId>ignore-failures</artifactId>
+  <version>1.0</version>
+  <packaging>pom</packaging>
+
+  <name>CasperJS Runner :: ignore-failures</name>
+
+  <url>no-url</url>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <casperjs.verbose>true</casperjs.verbose>
+  </properties>
+
+  <build>
+
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>com.github.casperjs</groupId>
+          <artifactId>casperjs-runner-maven-plugin</artifactId>
+          <version>@pom.version@</version>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+
+    <plugins>
+      <plugin>
+        <groupId>com.github.casperjs</groupId>
+        <artifactId>casperjs-runner-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>test</id>
+            <phase>integration-test</phase>
+            <goals>
+              <goal>test</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>verify</id>
+            <phase>verify</phase>
```

Comment:
specifying the phase will be useless once you add a defaultPhase to the Mojo annotation on the Verify goal (see below)